docs/releases/4.0.txt

```diff
@@ -213,6 +213,11 @@ Forms
   error message. This allows custom error messages to use the ``%(value)s``
   placeholder.
 
+* Fixed a bug in property ``BoundWidget.id_for_label`. It returned a wrong
+  ``id``, when used by a ``ChoiceField`` with ``widget=CheckboxSelectMultiple``
+  and rendered by a form initialized with ``auto_id='...%s...'``.
+  (:ticket:`32855`).
+
```

Comment:
I removed it.

However, since this changes the behavior of that widget (although only very few of us will notice this minor fix), I thought that it somehow shall be documented somewhere.